src/menus/interfaces.ts

```
@@ -0,0 +1,60 @@
+import * as Djs from "discord.js";
```

Biome (2.1.2)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)